Explorer/Assets/DCL/Landscape/Worlds/ChunkModel.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using Unity.Mathematics;
+using UnityEngine;
+
+namespace DCL.Landscape
+{
+    public class ChunkModel
+    {
+        public readonly int2 MinParcel;
+        public readonly int2 MaxParcel;
+
+        public readonly List<int2> OccupiedParcels;
```

Comment:
Can we expose these lists as `IReadOnly` to understand we are not going to write into them?